### PR TITLE
Add normalTxGasLimit

### PR DESCRIPTION
**Pull request by orenyodfat**
No need to use the overall gas limit (6300000) for all txs.
Use normalTxGasLimit for most txs

### CODE REVIEW

**Comment by dkent600** [comment]
@orenyodfat On what OS are you running this command?

(When I run it in a linux Bash shell on my machine it works fine.)

File: config/default.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "providerUrl": "http://127.0.0.1:8545",
   "network": "ganache",
   "gasLimit": 6300000,
```
Comment (dkent600) [inline config/default.json]: Please rename "gasLimit" to "gasLimit.deployment"

File: config/default.json [hunk 2]
```diff
@@ -2,6 +2,7 @@
...
+  "normalTxGasLimit": "4543760",
```
Comment (dkent600) [inline config/default.json]: Please rename "normalTxGasLimit" to "gasLimit.runtime"

File: config/default.json (hunk 1 above)
Comment (orenyodfat) [reply config/default.json]: done at 3f39eb4

**Comment by orenyodfat** [comment]
> @orenyodfat Have you tested these changes when running an app with Metamask against both ganache and kovan, including forging DAOs and doing other stuff?

No. do you have a test procedure for that ?

**Comment by dkent600** [comment]
@orenyodfat 

> No. do you have a test procedure for that ?

Nothing automated.  Right now we have to manually try it with Vanille or Alchemy and see what happens.

I think we need to do that to make sure we're not making things worse with these changes.  I will go ahead and do that since I think it will be a lot easier for me to do it.

**Comment by orenyodfat** [comment]
@dkent600 👍

**Comment by dkent600** [comment]
When with this PR:  https://github.com/daostack/arc.js/pull/127.

Created https://github.com/daostack/arc.js/issues/128 to follow up on forgeOrg

**Pull request closed without merging**